src/fp-paradigm/js/main.js

```diff
@@ -1,3 +1,4 @@
+// Looks wierd
 const state = {
     formGroups: {
         inputs: ['task'],
```

Comment:
If we talk about this side-effect, yes, because of that, i wrote 'look wierd', and need to be refactored.